package.json

```diff
@@ -214,6 +210,12 @@
     "untildify": "5.0.0",
     "valtio": "^1.13.2"
   },
+  "optionalDependencies": {
```

Comment:
I googled to see if there was a way to install these only on mac, but it looks like that's not a builtin npm feature. There's https://www.npmjs.com/package/npm-platform-dependencies, but it's archived and hasn't seen activity in almost a decade...

Reply:
- [ ] We probably need a script, maybe in postinstall, which will grab these deps if you're on a mac